app/controllers/charges_controller.rb

```diff
@@ -9,25 +9,36 @@ def new
   end
 
   def create
+    # I know this controller action is fat. It will be refactored in a future PR
     order = current_user.orders.pending.last
     if order
       # Amount in cents
       amount = order.stripe_total
-      customer = Stripe::Customer.create(
-        email:  current_user.email,
-        source: params[:stripeToken]
-      )
+
+      if session[:stripeToken]
+        customer = Stripe::Customer.create(
```

Comment:
Ah ok, I see what it is doing now. `stripe_token` is the `stripe_customer_id`. 
`customer_id = session[:stripeToken] ? create_stripe_customer.id : current_user.stripe_token`
Thanks, look forward to the future PR :)